docs/document/content/user-manual/shardingsphere-scaling/build.cn.md

```diff
@@ -50,13 +104,22 @@ mode:
 
 可以通过`ScalingClusterAutoSwitchAlgorithm`接口自定义一个SPI实现，通过`ScalingDataConsistencyCheckAlgorithm`接口自定义一个SPI实现。详情请参见[开发者手册#弹性伸缩](/cn/dev-manual/scaling/)。
 
+overwrite字段含义为：控制配置文件是否覆盖 zk 元数据，一般可在测试时使用。
```

Comment:
Not only for zk, could be any registry center.

Reply:
OK, get